ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouterFactory.java

```diff
@@ -112,6 +120,11 @@ public NonBlockingRouterFactory(VerifiableProperties verifiableProperties, Clust
     cryptoService = cryptoServiceFactory.getCryptoService();
     cryptoJobHandler = new CryptoJobHandler(routerConfig.routerCryptoJobsWorkerCount);
     defaultPartitionClass = clusterMapConfig.clusterMapDefaultPartitionClass;
+    FrontendConfig frontendConfig = new FrontendConfig(verifiableProperties);
+    AccountStatsStore accountStatsStore =
+        Utils.<AccountStatsStoreFactory>getObj(frontendConfig.accountStatsStoreFactory, verifiableProperties,
+            clusterMapConfig, new StatsManagerConfig(verifiableProperties),
+            clusterMap.getMetricRegistry()).getAccountStatsStore();
```

Comment:
what is this?

Reply:
Not sure why its here. Maybe due to rebase.